lib/src/config/dictionaries.rs

```diff
@@ -0,0 +1,175 @@
+use {
+    core::fmt,
+    std::{collections::HashMap, path::PathBuf},
+};
+
+#[derive(Clone, Debug, Eq, PartialEq, Hash)]
+pub struct DictionaryName(String);
+
+impl fmt::Display for DictionaryName {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        write!(f, "{}", self.0)
+    }
+}
+
+impl DictionaryName {
+    pub fn new(name: String) -> Self {
+        Self(name)
+    }
+}
+
+/// A single Dictionary definition.
+///
+/// A Dictionary consists of a name and an id, but more fields may be added in the future.
+#[derive(Clone, Debug)]
+pub struct Dictionary {
+    // Name must start with alphabetical and contain only alphanumeric, underscore, and whitespace
+    pub name: DictionaryName,
+    pub file: PathBuf,
+}
+
+/// A map of [`Backend`] definitions, keyed by their name.
```

Comment:
Copy-pasted doc comment needs updating.

Reply:
Updated in [`ff17124` (#61)](https://github.com/fastly/Viceroy/pull/61/commits/ff17124d03d8244ad1dfe77bfbaa79e44b55252b)